src/components/events/EventList.tsx

```diff
@@ -0,0 +1,73 @@
+import useSWR from 'swr'
+import { useRouter } from 'next/router'
+import Image from 'next/image'
+import { Linkbox } from 'hds-react'
+
+import { DrupalFormattedText } from '@/lib/types'
+import { getEvents } from '@/lib/client-api'
+
+import HtmlBlock from '@/components/HtmlBlock'
+import TagList from './TagList'
+import DateTime from './DateTime'
+
+import styles from './events.module.scss'
+
+interface EventListProps {
+  field_title: string
+  field_event_tag_filter: Array<string>
```

Comment:
For consistency we should decide if we are doing `Array<type>` or the shorthand version `type[]`. Both are valid, but we should choose one.